<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transmitter</class>
 <widget class="QWidget" name="transmitter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1507</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QComboBox" name="labelSelector">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="equipmentSelector">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [UKWN]</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Equipment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic [urw]</family>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Label:</string>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1501</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="headerCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="addLabel">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>10</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeLabel">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>10</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setupTransmitter">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>10</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Setup Transmitter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
